extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -201,6 +201,46 @@ In the following example, we display a `toolbar` if the window width is less tha
 See live demos at [AMP By Example](https://ampbyexample.com/components/amp-sidebar/).
 {% endcall %}
 
+## Sidebar for Stories
+The sidebar is also usable within the `amp-story` [component](https://www.ampproject.org/stories/). However there are certain restrictions on what attributes and features are allowed. The following are **disallowed** attributes and features of sidebar within an `amp-story`:
```

Comment:
"The sidebar is also usable within the `amp-story` [component](https://www.ampproject.org/stories/). "

to

"Use of `<amp-sidebar>` is supported within the `<amp-story>` [component](https://www.ampproject.org/stories/)."

Reply:
Done